docs/config.rst

```diff
@@ -35,18 +35,6 @@ Supported Formats
 
 php
 ---
-Configuration is declared in a variable $config
-
-    $config=[
```

Comment:
@romaninsh This was quite common syntax of config.php file in our old projects... but I guess it's better to return values and we can assign them to variable of out choice.